modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/api/AsyncResultSetImpl.java

```diff
@@ -41,6 +41,7 @@
  * Asynchronous result set implementation.
  */
 public class AsyncResultSetImpl<T> implements AsyncResultSet<T> {
+    @Nullable
```

Comment:
according to our guidelines, [type-use annotations appear immediately before the annotated type](https://google.github.io/styleguide/javaguide.html#s4.8.5-annotations)

Reply:
Fixed.